lib/internal/process/esm_loader.js

```diff
@@ -59,3 +61,19 @@ async function initializeLoader() {
     return exports.ESMLoader = ESMLoader;
   })();
 }
+
+exports.loadESM = async function loadESM(callback) {
+  try {
+    await initializeLoader();
```

Comment:
should probably gate this on `ESMLoader` being undefined

Reply:
do you mean add something like `assert(ESMLoader === undefined)` ?

Reply:
yeah that works.

Reply:
I can't. There's a default loader defined above, and removing it breaks things.